goc/config-osg/etc/cvmfs/domain.d/osgstorage.org.conf

```diff
@@ -4,7 +4,7 @@ if [ "$CVMFS_HTTP_PROXY" = "DIRECT" ]; then
 else
     CVMFS_SERVER_URL="http://cvmfs-s1bnl.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1fnal.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1goc.opensciencegrid.org/cvmfs/@fqrn@"
 fi
-CVMFS_EXTERNAL_URL="http://hcc-stash.unl.edu:8000/;http://osgxroot.usatlas.bnl.gov:8000/;http://stashcache.grid.uchicago.edu:8000/;http://xrd-cache-1.t2.ucsd.edu:8000/;http://mwt2-stashcache.campuscluster.illinois.edu:8000/;http://its-condor-xrootd1.syr.edu:8000/"
+CVMFS_EXTERNAL_URL="http://hcc-stash.unl.edu:8000/;http://osgxroot.usatlas.bnl.gov:8000/;http://stashcache.grid.uchicago.edu:8000/;http://xrd-cache-1.t2.ucsd.edu:8000/;http://mwt2-stashcache.campuscluster.illinois.edu:8000/;http://its-condor-xrootd1.syr.edu:8000/;http://xcache-11.t2.ucsd.edu:8000;http://gpn-fiona-r740xd-node809.greatplains.net:8000"
```

Comment:
Are there really 2 servers at ucsd.edu?  I wonder if the new one was supposed to replace the old one.  They aren't named similarly.  If they're both to be used in production, I would recommend putting them together on the list so it's easier to see at a glance that they're together.  The geo sorting will not be able to distinguish them so I think the one listed first will get all the traffic and the second one will only be contacted if the first one fails.  It would probably be better to put them both into a DNS round robin if they want both of them to get used.

I don't know if it's important, but all the other URLs are listed with a trailing slash so we should probably be consistent.

The node for the greatplains.net node is certainly not very generic.  I think it would be worth a brief pushback to see if they'd be willing to give it a DNS alias, so they could change nodes in the future without making osg (you) do another update to the config repo.